apps/i18n/tutorialExplorer/en_us.json

```diff
@@ -49,5 +49,6 @@
   "tutorialDetailsShortLink": "Short link",
   "tutorialDetailInternationalLanguages": "Languages",
   "headingTutorialsYourLanguage": "Tutorials in your language",
-  "headingTutorialsManyLanguages": "Tutorials in many languages"
+  "headingTutorialsManyLanguages": "Tutorials in many languages",
+  "tutorialDetailStandards": "Standards"
```

Comment:
elsewhere we've tried to keep our strings mostly alphabetized. might be worth doing that here too.